__tests__/pages/webhooks/order-created.test.ts

```diff
@@ -0,0 +1,182 @@
+/** @jest-environment setup-polly-jest/jest-environment-node */
+
+import { PollyServer } from "@pollyjs/core";
+import * as joseModule from "jose";
+import { NextApiRequest, NextApiResponse } from "next";
+import toNextHandler from "../../../pages/api/webhooks/order-created";
+import { setupPollyMiddleware, setupRecording } from "../../pollySetup";
+import {
+  dummyOrderCreatedPayload,
+  mockRequest,
+  dummyFetchTaxesResponse,
+} from "../../utils";
+import Avatax from "ava-typescript";
+
+jest.mock("next/dist/compiled/raw-body/index.js", () => ({
+  __esModule: true,
+  default: jest.fn((_) => ({
+    toString: () => '{"dummy":12}',
+  })),
+}));
```

Comment:
*please ignore if that's too much work, I'm just leaving it so we can improve that mocking later*

@mmiszy wrote a nice utility function for mocking requests with `bodyParser` disabled: https://github.com/saleor/react-storefront/blob/main/apps/saleor-app-checkout/test-utils.ts#L36